zerver/models.py

```diff
@@ -3891,13 +3902,19 @@ def is_cross_realm_bot_email(email: str) -> bool:
     return email.lower() in settings.CROSS_REALM_BOT_EMAILS
 
 
-# The Huddle class represents a group of individuals who have had a
-# group private message conversation together.  The actual membership
-# of the Huddle is stored in the Subscription table just like with
-# Streams, and a hash of that list is stored in the huddle_hash field
-# below, to support efficiently mapping from a set of users to the
-# corresponding Huddle object.
 class Huddle(models.Model):
+    """
+    The Huddle class represents a group of individuals who have had a
+    group private message conversation together.
```

Comment:
Strictly speaking, a `Huddle` can be created just by a `get_huddle` call - so you can have Huddle of users that did not actually have conversation together. So we could rephrase this sentence to be more correct - but I'm not sure if that level of subtlety would really be helpful

Reply:
That does happen in the typing notifications code path, I think.

Reply:
Hmm, I've changed this to `The Huddle class represents a group of individuals who may have a group private message conversation together.` so that there doesn't appear to be an implication that a `Huddle` existing for users means they've exchanged messages. Perhaps that's better?